week2/task4/ex4.cs

```diff
@@ -0,0 +1,95 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Excercise-4!");
+var time = new JupiterTime(2, 20);
+time.AddMinutes(-20);
+PrintTime(time);
+
+
+void PrintTime(JupiterTime time)
+{
+
+    Console.WriteLine($"{time.Hours}:{time.Minutes}");
+
+
```

Comment:
Try to avoid using new lines if they are not used to splitting functionality or logic